src/containers/RepositoriesContainer/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './RepositoriesContainer.js';
```

Comment:
Haha yeah I was pretty pleased when I came across this too. Will tidy up the others to use it this week too :D